### PR TITLE
fix: Updated dependencies to clear vulnerabilities

### PR DESCRIPTION
**Pull request by bparmar-splunk**

### CODE REVIEW

**Comment by VihasMakwana** [comment]
LGTM.
Please sync this with develop and I'll merge it after pipeline passed

**Review by VihasMakwana — DISMISSED** [review]

**Comment by bparmar-splunk** [comment]
> LGTM. Please sync this with develop and I'll merge it after pipeline passed

I have synced my fork. It is pending for your approval.

**Comment by VihasMakwana** [comment]
Fixes: https://github.com/splunk/kafka-connect-splunk/issues/380

**Pull request merged**
